plenary/plenary-30.yaml

```diff
@@ -0,0 +1,226 @@
+metadata:
+  title: Resolutions from the 30th plenary meeting of ISO/TC 154
+  date: 2011-09-14
+  source: ISO/TC 154 Secretariat
+resolutions:
+  - dates:
+      - 2011-09-14
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 310
+    
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved unanimously, noting the Abstention from the UK
+
+    actions:
+      - type: resolves
+        date_effective: 2011-09-14
+        message: instruct its secretariat to take all the necessary steps to establish a formal liaison with ETSI TC “Electronic Signatures and Infrastructures” (ESI);
+      - type: resolves
+        date_effective: 2011-09-14
+        message: instruct its secretariat to ballot ETSI TC ESI’s NWIP in TC 154.
+
+  - dates:
+      - 2011-09-14
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 311
+    
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved unanimously
+
+    actions:
+      - type: resolves
+        date_effective: 2011-09-14
+        message: ISO/TC 154 resolves to accept the NWIP to update ISO 9735-10:2002 taking into account all releases of the service code list directories starting from release 40006 until the current release
+        
+    # Note: The NWIP is available in the meeting documents.
+
+
+  - dates:
+      - 2011-09-14
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 312
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved unanimously
+
+    actions:
+      - type: resolves
+        date_effective: 2011-09-14
+        message: Accept the latest draft of ISO/NP TS 17369 as the deliverable of WG2
+      - type: resolves
+        date_effective: 2011-09-14
+        message: Dissolve WG2
+
+
+  - dates:
+      - 2011-09-14
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 313
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved unanimously
+        
+    actions:
+      - type: resolves
+        date_effective: 2011-09-14
+        message: |
+        ISO/TC 154 resolves to:
+        - Acknowledge the deliverable of ISO/NP TS 17369 to become a full ISO standard;
+        - Accept the latest draft delivered by WG2 as a Committee Draft;
+        - Install an ad-hoc working group with the task to deliver a DIS 17369 to the TC154 secretariat and invite TC 154 members to participate;
+        - Install Mr Chris Nelson from the United Kingdom as Project leader
```

Comment:
Is it correct to put several `resolves` statements in one message like above?

Original source:
![capture6](https://user-images.githubusercontent.com/33627611/71759196-c0e24500-2e7f-11ea-8588-cd287073f8ed.JPG)

Reply:
Ideally we should encode the 4 `resolves` as separate actions, but we also need a place to store the original message...

So let's add a higher level `message:` key to store the original message, and then break it down to "acknowledges", "accepts", "creates", "appoints". (see new code)